grammars/pegcoffee.cson

```diff
@@ -62,15 +62,13 @@
 'repository':
   'charSet':
     'begin': '\\['
-    'end': '\\](i?)'
     'beginCaptures':
       '0':
         'name': 'keyword.operator'
+    'end': '\\](i?)'
     'endCaptures':
       '0':
         'name': 'keyword.operator'
-      '1':
```

Comment:
Why was this removed?

Reply:
We can have `[a-z]*` or `[a-z]+` or `[a-z]i`. No real reason to treat `i` differently than `*` and `+`.

Reply:
Makes sense :+1: